packages/astrocite-bibtex/src/parser.ts

```diff
@@ -52,8 +53,20 @@ const parseValue = (
                 output += `${KNOWN_COMMANDS.get(v.value) || ''}`;
                 break;
             case 'DicraticalCommand':
-                output += `${v.character}${DICRATICS.get(v.mark)}`;
+                char =
+                    typeof v.character === 'string'
+                        ? v.character
+                        : v.character.character;
+                output += `${char}${DICRATICS.get(v.mark)}`;
                 break;
+            case 'SuperscriptCommand':
+            case 'SubscriptCommand':
+                output += v.value;
+                break;
+            case 'MathMode':
+                break;
+            default:
+                output += JSON.stringify(v);
```

Comment:
When does the default case run? Do you have an example of that on hand? Seems strange to dump stringified JSON in the output. I would think that we'd just skip over it if it doesn't match any of the named cases.

Reply:
Damn, sorry, that should not have been left there.